src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleaseHelpAndSupportSection.tsx

```diff
@@ -89,12 +89,18 @@ const AccordionComponent = ({
   children,
 }: AccordionComponentProps) => {
   return includeAnalytics ? (
-    <AccordionWithAnalytics
```

Comment:
I said get rid of `AccordionWithAnalytics` completely, don't think there's any change for that.